layers/+emacs/org/packages.el

```diff
@@ -1003,3 +1004,11 @@ Headline^^            Visit entry^^               Filter^^                    Da
       (setq org-appear-autolinks t
             org-appear-autoemphasis t
             org-appear-autosubmarkers t))))
+
+(defun org/init-org-roam-server()
+  (use-package org-roam-server
+    :defer t
+    :init
+    (progn
+      (spacemacs/set-leader-keys "aors" 'org-roam-server-mode)
+      (spacemacs/set-leader-keys-for-major-mode 'org-mode "mrs" 'org-roam-server-mode))))
```

Comment:
When you use `set-leader-keys-for-major-mode` it will automatically get bound behind `SPC m` / `,`, the binding will be available only in org-mode buffers (remove `m`)